Dokan/Dokan/Scenes/ProductDetails/ProductDetailsViewController.xib

```diff
@@ -110,4 +140,12 @@
             <point key="canvasLocation" x="91.304347826086968" y="219.30803571428569"/>
         </view>
     </objects>
+    <resources>
+        <systemColor name="labelColor">
+            <color white="0.0" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+        <systemColor name="systemBackgroundColor">
+            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+    </resources>
```

Comment:
We don't update styles in XIB, we should fall back to the design system practices.